dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
```

Comment:
Can you add python3_9 ?-)

Reply:
python3.9 is still in beta.  repoman gives an error when including it in ebuild.

Reply:
If the error is along the lines of "Invalid implementation in PYTHON_COMPAT: python3_9", try syncing your fork with :gentoo

Reply:
Looks to me like python3.9 works in this one. Rebase your branch and force-push.